src/theme/components/Button.js

```diff
@@ -380,7 +380,8 @@ export default (variables /* : * */ = variable) => {
     shadowOpacity: platformStyle === PLATFORM.MATERIAL ? 0.2 : undefined,
     shadowRadius: platformStyle === PLATFORM.MATERIAL ? 1.2 : undefined,
     alignItems: 'center',
-    justifyContent: 'space-between'
+    justifyContent: 'space-between',
+    alignSelf: 'flex-start'
```

Comment:
Can we do it without using alignSelf? The alignment of the button should be left to the user.

Reply:
I think `alignSelf` is fine. I don't find any other solution. The alignment of the button is left to the user. Isn't it?

Reply:
I mean the user might want to align it to the center or to the right. So I think it should be the user's choice, can we use `alignSelf: 'auto'` and see if that works?

Reply:
`alignSelf: 'auto' `will set width to full. But User always has a choice for `alignSelf: 'center'` whether we set `alignSelf: 'flex-start'` by default or not.